### PR TITLE
feat(docker): Explain how to configure a postgres container for bonita

### PR DESCRIPTION
**Pull request by danila-m**
Relates to [RUNTIME-544](https://bonitasoft.atlassian.net/browse/RUNTIME-544)

### CODE REVIEW

File: modules/ROOT/pages/bonita-docker-installation.adoc [hunk 1]
```diff
@@ -112,19 +112,24 @@ You can work with either a PostgreSQL Container, or PostgreSQL as an installed s
...
 
-----
-docker run --name mydbpostgres -e POSTGRES_PASSWORD=mysecretpassword -d postgres:11.2 -c 'max_prepared_transactions=100'
-----
+From Bonita 2022.1 onwards, the Bonita docker image does not include configuration scripts for Postgres
+====
 
-See the official PostgreSQL documentation for more details.
+Therefore the PostgreSQL container needs to be configured to work with Bonita before starting the Bonita container.
+The configuration of a PostgreSQL database to work with Bonita is described in detail in the xref:database-configuration.adoc[database configuration page]
```
Comment (educhastenier) [inline modules/ROOT/pages/bonita-docker-installation.adoc]:
```suggestion
The configuration of a PostgreSQL database to work with Bonita is described in detail in the xref:database-configuration.adoc[database configuration page]. +
```

File: modules/ROOT/pages/bonita-docker-installation.adoc [hunk 2]
```diff
@@ -112,19 +112,24 @@ You can work with either a PostgreSQL Container, or PostgreSQL as an installed s
...
+Alternatively, Bonita provides a preconfigured https://hub.docker.com/repository/docker/bonitasoft/bonita-postgres[postgres image] on docker-hub
```
Comment (educhastenier) [inline modules/ROOT/pages/bonita-docker-installation.adoc]:
```suggestion
Alternatively, Bonita provides a preconfigured https://hub.docker.com/r/bonitasoft/bonita-postgres[PostgreSQL image] on docker-hub. +
```

File: modules/ROOT/pages/bonita-docker-installation.adoc [hunk 3]
```diff
@@ -112,19 +112,24 @@ You can work with either a PostgreSQL Container, or PostgreSQL as an installed s
...
+You can run the image with the following command
```
Comment (educhastenier) [inline modules/ROOT/pages/bonita-docker-installation.adoc]:
```suggestion
You can run the image with the following command:
```

File: modules/ROOT/pages/bonita-docker-installation.adoc [hunk 4]
```diff
@@ -112,19 +112,24 @@ You can work with either a PostgreSQL Container, or PostgreSQL as an installed s
...
 
-[source,shell script,subs="+macros"]
 ----
-docker run --name bonita_postgres -h <hostname> -v bonita-lic:/opt/bonita_lic/ --link mydbpostgres:postgres -d -p 8080:8080 quay.io/bonitasoft/bonita-subscription:pass:a[{bonitaVersion}]
+docker run --name mydbpostgres -h <hostname> -v bonita-lic:/opt/bonita_lic/ -d bonitasoft/bonita-postgres:12.6
```
Comment (educhastenier) [inline modules/ROOT/pages/bonita-docker-installation.adoc]:
```suggestion
docker run --name mydbpostgres -h <hostname> -v <your-bonita-license-folder>:/opt/bonita_lic/ -d bonitasoft/bonita-postgres:12.6
```

**Review by educhastenier — CHANGES_REQUESTED** [review]
Also modify docker-compose yaml example from
image: postgres:11.2
to
image: postgres:12.6

**Comment by baptistemesta** [comment]
there are reference to ENSURE_DB_CHECK_AND_CREATION here: https://documentation.bonitasoft.com/bonita/2021.2/bonita-docker-installation#_ensure_db_check_and_creation

**Review by educhastenier — APPROVED** [review]

**Pull request merged**